openshift/odo
# Implementation of a walker for smart odo push

fixes #1788 

How to test:

- run `odo push`
- a `odo-file-index.json` will be generate with all the fileNames, sizes and lastModifiedDates
- rerun and check if the build gets started or not
- use `-f` flag to force build
